packages/twenty-front/src/modules/auth/sign-in-up/hooks/useSSO.ts

```diff
@@ -0,0 +1,66 @@
+import { isDefined } from '~/utils/isDefined';
+import { SnackBarVariant } from '@/ui/feedback/snack-bar-manager/components/SnackBar';
+import { useSnackBar } from '@/ui/feedback/snack-bar-manager/hooks/useSnackBar';
+import {
+  FindAvailableSsoIdentityProvidersMutationVariables,
+  GetAuthorizationUrlMutationVariables,
+  useFindAvailableSsoIdentityProvidersMutation,
+  useGetAuthorizationUrlMutation,
+} from '~/generated/graphql';
+
+export const useSSO = () => {
+  const { enqueueSnackBar } = useSnackBar();
+
+  const [findAvailableSSOProviderByEmailMutation] =
+    useFindAvailableSsoIdentityProvidersMutation();
```

Comment:
is it a mutation or a query?

Reply:
We need to send an email to the backend. Can I do it with a query?